dali/operators/generic/constant.cc

```diff
@@ -25,41 +25,49 @@ namespace dali {
 DALI_SCHEMA(Constant)
   .DocStr(R"code(Produces a batch of constant tensors.
 
-The floating point input data should be placed in `fdata` argument and integer data in `idata`.
-The data is a flat vector of values or a single scalar. The data is then reshaped according
-to the `shape` argument. If the data is scalar, it will be broadcast to fill the entire shape.
+The floating point input data should be placed in the ``fdata`` argument
+and integer data in ``idata``. The data, which can be a flat vector of values or a scalar,
+is then reshaped according to the ``shape`` argument. If the data is scalar, it will be broadcast
+to fill the entire shape.
 
-The operator only performs meaningful work at first invocation - subsequent calls will return
+The operator only performs meaningful work at first invocation; subsequent calls will return
 a reference to the same memory.
 
-The operator can be automatically instantiated in Python with a call to
+To automatically instantiate the operator in Python, call
```

Comment:
Calling `types.Constant` does not always create Constant operator - sometimes it creates a ScalarConstant object - so this sentence is incorrect.
I have no better suggestion than to revert
```suggestion
he operator can be automatically instantiated in Python with a call to
```

Reply:
Done